lib/vdsm/api/vdsm-api.yml

```diff
@@ -12082,6 +12082,14 @@ Volume.measure:
           or the entire chain.
         name: backing_chain
         type: boolean
+
+    -   defaultvalue: null
+        description: If specified, the base volume UUID, measuring the
```

Comment:
This is IMHO hard to understand. Maybe something like "If the baseID is specified, only the sub-chain will be measured. The range of sub-chain is from volumeID to baseID. Currently the only allowed value of baseID is parent volume UUID."?

Reply:
I will use your text, thanks!